### PR TITLE
 imx-mcore-demos: Fix install for multilib  Replace  `base_libdir` with `nonarch_base_libdir`

### PR DESCRIPTION
**Pull request by valijec**
When using multilib configuration there is a build break at packaging step

### CODE REVIEW

File: recipes-fsl/mcore-demos/imx-mcore-demos.inc [hunk 1]
```diff
@@ -57,6 +57,6 @@ addtask deploy after do_install
 
 PACKAGE_ARCH = "${MACHINE_SOCARCH}"
 
-FILES:${PN} = "${nonarch_base_libdir}/firmware"
+FILES:${PN} = "${baselib}/firmware"
```
Comment (thochstein) [inline recipes-fsl/mcore-demos/imx-mcore-demos.inc:60]: It's `do_install` that needs to be fixed, installing to `nonarch_base_libdir` instead.

**Review by thochstein — CHANGES_REQUESTED** [review]
It's `do_install` that needs to be fixed, installing to `nonarch_base_libdir` instead.

File: recipes-fsl/mcore-demos/imx-mcore-demos.inc (hunk 1 above)
Comment (valijec) [reply recipes-fsl/mcore-demos/imx-mcore-demos.inc]: Thank you @thochstein, fixed that

File: recipes-fsl/mcore-demos/imx-mcore-demos.inc [hunk 2]
```diff
@@ -40,9 +40,9 @@ S = "${WORKDIR}/${SOC}-${MCORE_TYPE}-demo-${PV}"
 SCR = "SCR-${SOC}-${MCORE_TYPE}-demo.txt"
 
 do_install () {
-    # install elf format binary to /lib/firmware
-    install -d ${D}${base_libdir}/firmware
-    install -m 0644 ${S}/*.elf ${D}${base_libdir}/firmware
+    # install elf format binary to /usr/lib/firmware
```
Comment (quaresmajose) [inline recipes-fsl/mcore-demos/imx-mcore-demos.inc:43]: It will be better to drop this line as it depends on multilib or usermerge

**Review by thochstein — APPROVED** [review]

**Pull request merged**